kai/import_from_hub.py

```diff
@@ -0,0 +1,338 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import pprint
+import subprocess
+import tempfile
+import time
+from typing import Any, Dict, List, Optional, Tuple
+
+import dateutil.parser
+import requests
+import urllib3
+from pydantic import BaseModel, Field, HttpUrl
+
+from kai.kai_logging import KAI_LOG
+from kai.models.kai_config import KaiConfig
+from kai.report import Report
+from kai.service.incident_store import Application, IncidentStore
+
+
+# BaseModel that also acts as a dict
+class KaiBaseModel(BaseModel):
+    def __contains__(self, item):
+        return hasattr(self, item)
+
+    def __getitem__(self, key):
+        if not hasattr(self, key):
+            raise KeyError(f"Key '{key}' not found")
+        return getattr(self, key)
+
+    def __setitem__(self, key, value):
+        if not isinstance(key, str):
+            raise ValueError("Key must be a string")
+        setattr(self, key, value)
+
+    def get(self, key, default=None):
+        if key in self:
+            return self[key]
+        return default
+
+
+class Incident(KaiBaseModel):
```

Comment:
Should we think about removing [analyzer_types.py](https://github.com/konveyor-ecosystem/kai/blob/main/kai/models/analyzer_types.py) if we have these instead? I think it'd be good to consolidate the models

Reply:
Yeah maybe, these models correspond to the konveyor API types though which are slightly different from the types in the report itself.